docs_src/hardwaresetup.md

```diff
@@ -27,17 +28,6 @@ Click the links for corresponding set up instructions.
 
     3. [Set up the pipeline](./OVMS/pipelinesetup.md)
 
-=== "Intel® Xeon® Processor"
-    1. Download [Ubuntu 22.04](https://releases.ubuntu.com/22.04/) and follow these [installation steps](https://ubuntu.com/tutorials/install-ubuntu-desktop#1-overview).
-
-    2. Install [Docker Engine](https://docs.docker.com/engine/install/ubuntu/)
-
-    3. [**Optional**] Install [Docker Compose v2](https://docs.docker.com/compose/), if using the docker compose feature
-    
-    4. Install [Git](https://git-scm.com/download/linux)
-
-    3. [Set up the pipeline](./OVMS/pipelinesetup.md)
-
 === "Intel® Data Center GPU Flex 140/170"
     1. Download [Ubuntu 22.04](https://releases.ubuntu.com/22.04/) and follow these [installation steps](https://ubuntu.com/tutorials/install-ubuntu-desktop#1-overview).
```

Comment:
have we figured out the deprecated html page  issue?

Reply:
this https://dgpu-docs.intel.com/installation-guides/ubuntu/ubuntu-jammy-dc.html page is deprecated.

Reply:
You can point to https://dgpu-docs.intel.com/driver/installation.html#ubuntu-install-steps